src/core/path.h

```diff
@@ -67,6 +101,21 @@ typedef struct QUIC_PATH {
     //
     uint8_t PartitionUpdated : 1;
 
+    //
+    // ECN validation state.
+    //
+    uint8_t EcnValidationState : 2;
+
+    //
+    // Indicates whether EcnTestingEndingTime is set.
+    //
+    BOOLEAN EcnTestingEndingTimeSet : 1;
```

Comment:
Can you just consider a value of `0` for `EcnTestingEndingTime` as unset?

Reply:
I thought about this but I don't know if this is the convention, using a possible value as sentinel, in QUIC.

Reply:
Let's just use `0`. For 64-bit time values zero is never valid.

Reply:
updated